api/src/room_store/redis_room_listener.py

```diff
@@ -157,5 +157,5 @@ async def create_redis_room_listener(redis: Redis) -> AsyncIterator[RedisRoomLis
         try:
             yield listener
         finally:
-            await pubsub.reset()
+            await pubsub.aclose()
```

Comment:
The deprecation instructions explicitly say to replace reset with aclose, I know it sounds weird.